tests/rbd/rbd_migration_2diff_ec_pool_ec.py

```diff
@@ -0,0 +1,158 @@
+    srcpool1 = config_data["config_1"]["ec_pool_config"]["pool"]
+    datapool1 = config_data["config_1"]["ec_pool_config"]["data_pool"]
+    src_spec1 = srcpool1 + "/" + image1
+
+    image2 = config_data["config_2"]["ec_pool_config"]["image"]
+    srcpool2 = config_data["config_2"]["ec_pool_config"]["pool"]
+    datapool2 = config_data["config_2"]["ec_pool_config"]["data_pool"]
+    src_spec2 = srcpool2 + "/" + image2
+
+    destpool = config_data["config_3"]["ec_pool_config"]["pool"]
+    datapool3 = config_data["config_3"]["ec_pool_config"]["data_pool"]
+    dest_spec1 = destpool + "/" + image1
+    dest_spec2 = destpool + "/" + image2
+
+    try:
+        client = kw["ceph_cluster"].get_nodes(role="client")[0]
+        with parallel() as p:
+            p.spawn(run_fio, image_name=image1, pool_name=srcpool1, client_node=client)
+            p.spawn(run_fio, image_name=image2, pool_name=srcpool2, client_node=client)
+        rbd = Rbd(**kw)
+        # Perpare the migration process from 2 source pool to destination pool
+        with parallel() as p:
+            p.spawn(rbd.migration_prepare, src_spec1, dest_spec1)
+            p.spawn(rbd.migration_prepare, src_spec2, dest_spec2)
+
+        # verify migration prepare
+        with parallel() as p:
+            p.spawn(verify_migration_state, rbd, dest_spec1)
+            p.spawn(verify_migration_state, rbd, dest_spec2)
+
+        # run io's on the destination image
+        with parallel() as p:
+            p.spawn(run_fio, image_name=image1, pool_name=destpool, client_node=client)
+            p.spawn(run_fio, image_name=image2, pool_name=destpool, client_node=client)
+
+        # Execute migration process
+        with parallel() as p:
+            p.spawn(rbd.migration_action, action="execute", dest_spec=dest_spec1)
+            p.spawn(rbd.migration_action, action="execute", dest_spec=dest_spec2)
+
+        # verify execute migration process
+        with parallel() as p:
+            p.spawn(verify_migration_state, rbd, dest_spec1)
+            p.spawn(verify_migration_state, rbd, dest_spec2)
+
+        # commit migration process
+        with parallel() as p:
+            p.spawn(rbd.migration_action, action="commit", dest_spec=dest_spec1)
+            p.spawn(rbd.migration_action, action="commit", dest_spec=dest_spec2)
+
+        # verify commit migration
+        with parallel() as p:
+            p.spawn(verify_migration_commit, rbd, destpool, image1)
+            p.spawn(verify_migration_commit, rbd, destpool, image2)
+
+        images = [image1, image2]
+        # Display image info and verify
+        for image in images:
+            image_info = rbd.image_info(destpool, image)
+            log.info(f"Image info : {image_info}")
+            if image not in image_info["name"]:
+                log.error(
+                    f"Image {image} is not listed in 'rbd image info' for pool {destpool}"
+                )
+                return 1
```

Comment:
I dont think we need this block, since rbd.image_info is executed in above calls.

Reply:
as per suggestion Removed the code block.